src/pipelines/wasmbg.rs

```diff
@@ -67,6 +70,30 @@ impl WasmBindgen {
             String::from_utf8_lossy(&build_output.stderr),
         );
 
+        // wasm-bindgen build succeeded so delete everything in `dist`,
+        // copy everything from `dist/.current` to `dist` and
+        // delete `dist/.current`
+        let mut entries = fs::read_dir(self.bindgen_out.as_path()).await?;
+        while let Some(res) = entries.next().await {
+            let entry = res?;
+            if entry.path().to_string_lossy() != bindgen_out.to_string_lossy() {
+                let path = entry.path();
+                if path.is_dir().await {
+                    fs::remove_dir_all(&path).await.context("Deleting directory after build failed")?
```

Comment:
It isn't reliable on Windows. We should use something like https://github.com/XAMPPRocky/remove_dir_all.
See https://blog.qwaz.io/chat/issues-of-rusts-remove-dir-all-implementation-on-windows for more info.

Reply:
@MartinKavik do you know if that applies to just the `std` implementation, or also to the [async_std::remove_dir_all](https://docs.rs/async-std/1.8.0/async_std/fs/fn.remove_dir_all.html) implementation?

Reply:
Looking at the async-std source code (https://github.com/async-rs/async-std/blob/master/src/fs/remove_dir_all.rs#L31-L38), it doesn't seem to do that. It just uses `std::fs` implementation directly. So I don't think it would work in `async-std` either